lib/reports/ppl-conditions/parse.js

```diff
@@ -1,5 +1,7 @@
-const { omit, concat, flatten, pick } = require('lodash');
+const { omit, get, concat, pick } = require('lodash');
 const moment = require('moment');
+const CONDITIONS_SPEC = require('@asl/projects/client/constants/conditions').default;
+const RA = require('@asl/projects/client/constants/retrospective-assessment').default;
 
 const getCondition = (condition, level, protocolTitle = '') => {
   return {
```

Comment:
in the original code it was 

nst defaults = {
   level: '',
   protocol_name: '',
   type: '',
   condition: 'none',
   requires_editing: '',
   edited: '',
   content: ''
 };
 
 whereas yours is only a subset?

Reply:
This is the "getCondition" function that was originally in this file that normalises the conditions from the project/protocol data. See "conditionToReportRow" for the mapper from asl-internal-ui - that should have the same behaviour, except it's not including some properties that was being implicitly ignored before due to the way that the CSV mapper uses the first entry to determine the headers (hence omit -> pick).

I will rename this so it's clearer what it's for though.